Formula/llvm.rb

```diff
@@ -188,17 +226,36 @@ def install
     args = %w[
       -DLLVM_OPTIMIZED_TABLEGEN=On
       -DLLVM_BUILD_LLVM_DYLIB=On
+      -DLLVM_TARGETS_TO_BUILD=X86
```

Comment:
Two reasons:
1. `all` means: `AArch64, AMDGPU, ARM, BPF, Hexagon, Mips, MSP430, NVPTX, PowerPC, Sparc, SystemZ, X86, XCore`.  -- too many for test runs.
2. Only rare developers will need all of them at once. Plus, Homebrew has target audience. So, perhaps it can/should be extended to: `X86;AMDGPU;NVPTX`. Not sure if `PowerPC` is needed.